pkg/k8s/utils/utils.go

```diff
@@ -219,3 +220,15 @@ func GetClusterIPByFamily(ipFamily slim_corev1.IPFamily, service *slim_corev1.Se
 
 	return ""
 }
+
+// EscapeJsonPointer escapes string to be used as a JSON pointer.
+//
+// This is performed by first transforming any
+//
+//	occurrence of the sequence '~' to '~0', and then transforming any
+//	occurrence of the sequence '/' to '~1'.
+//
+// Details - https://datatracker.ietf.org/doc/html/rfc6901#section-4
+func EscapeJsonPointer(s string) string {
```

Comment:
I'm not seeing this used anywhere.

Reply:
I assume that someone may use it oneday. Ok then, deleted.